frontend/src/utils/generate-github-auth-url.ts

```diff
@@ -7,6 +7,19 @@
  */
 export const generateGitHubAuthUrl = (clientId: string, requestUrl: URL) => {
   const redirectUri = `${requestUrl.origin}/oauth/keycloak/callback`;
+
+  // Check if GitHub Enterprise Server URL is configured
+  const githubEnterpriseUrl = window.GITHUB_ENTERPRISE_URL || "";
```

Comment:
can we have a param from the user settings passed to this function instead of pulling the URL value out from `window`